src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -624,14 +624,14 @@ def test_generate_schedule_unmatching_arguments(parser):
     # Test exception for unmatching argument list
```

Comment:
Please could you fix the two `useless-object-inheritance` warnings and the two unused imports.

Reply:
Done (for all object subclassing in src/)

Reply:
Thanks but the two unused imports are still there.

Reply:
Ups, done now.